test/unittests/backends/webgl/test_concat_packed.ts

```diff
@@ -0,0 +1,279 @@
+    //   outputTextureShape: [2, 1],
+    //   useGeneratedOutput: true,
+    // },
+    // {
+    //   elementCount: 6,
+    //   axis: 0,
+    //   inputShape: [2, 3],
+    //   outputShape: [2, 3],
+    //   inputTextureShape: [2, 1],
+    //   outputTextureShape: [2, 3],
+    //   rawData: new Float32Array([1, 2, 4, 5, 3, 0, 6, 0]),
+    // },
+    // {
+    //   elementCount: 6,
+    //   axis: 1,
+    //   inputShape: [2, 3],
+    //   outputShape: [2, 3],
+    //   inputTextureShape: [2, 1],
+    //   outputTextureShape: [2, 3],
+    //   rawData: new Float32Array([1, 2, 4, 5, 3, 0, 6, 0]),
+    // },
+
+    // // // test 3d tensor
+    // {
+    //   elementCount: 16,
+    //   axis: 0,
+    //   inputShape: [2, 2, 4],
+    //   outputShape: [2, 2, 4],
+    //   inputTextureShape: [2, 2],
+    //   outputTextureShape: [4, 4],
+    //   useGeneratedOutput: true,
+    // },
+    // {
+    //   elementCount: 16,
+    //   axis: 1,
+    //   inputShape: [2, 2, 4],
+    //   outputShape: [2, 2, 4],
+    //   inputTextureShape: [2, 2],
+    //   outputTextureShape: [4, 4],
+    //   useGeneratedOutput: true,
+    // },
+    // {
+    //   elementCount: 24,
+    //   axis: 0,
+    //   inputShape: [2, 3, 4],
+    //   outputShape: [2, 3, 4],
+    //   inputTextureShape: [2, 4],
+    //   outputTextureShape: [6, 4],
+    //   rawData: new Float32Array([
+    //     1,  2,  5,  6,  3,  4,  7,  8,  9,  10, 0, 0, 11, 12, 0, 0,
+    //     13, 14, 17, 18, 15, 16, 19, 20, 21, 22, 0, 0, 23, 24, 0, 0
+    //   ])
+    // },
+    // {
+    //   elementCount: 24,
+    //   axis: 1,
+    //   inputShape: [2, 3, 4],
+    //   outputShape: [2, 3, 4],
+    //   inputTextureShape: [2, 4],
+    //   outputTextureShape: [6, 4],
+    //   rawData: new Float32Array([
+    //     1,  2,  5,  6,  3,  4,  7,  8,  9,  10, 0, 0, 11, 12, 0, 0,
+    //     13, 14, 17, 18, 15, 16, 19, 20, 21, 22, 0, 0, 23, 24, 0, 0
+    //   ])
+    // },
+    // // test 4d tensor
+    // {
+    //   elementCount: 32,
+    //   axis: 0,
+    //   inputShape: [2, 2, 2, 4],
+    //   outputShape: [2, 2, 2, 4],
+    //   inputTextureShape: [2, 4],
+    //   outputTextureShape: [8, 4],
+    //   useGeneratedOutput: true,
+    // },
+    // {
+    //   elementCount: 32,
+    //   axis: 1,
+    //   inputShape: [2, 2, 2, 4],
+    //   outputShape: [2, 2, 2, 4],
+    //   inputTextureShape: [2, 4],
+    //   outputTextureShape: [8, 4],
+    //   useGeneratedOutput: true,
+    // },
+    // {
+    //   elementCount: 64,
+    //   axis: 0,
+    //   inputShape: [2, 2, 4, 4],
+    //   outputShape: [2, 2, 4, 4],
+    //   inputTextureShape: [2, 8],
+    //   outputTextureShape: [16, 4],
+    //   useGeneratedOutput: true,
+    // },
+    // {
+    //   elementCount: 64,
+    //   axis: 1,
+    //   inputShape: [2, 2, 4, 4],
+    //   outputShape: [2, 2, 4, 4],
+    //   inputTextureShape: [2, 8],
+    //   outputTextureShape: [16, 4],
+    //   useGeneratedOutput: true,
+    // },
```

Comment:
Should these be re-enabled?

Reply:
was trying to see how the max CI would react to concat, looks like it didnt like that either...  added the check to skip on webgl1 and re-enabled other tests.